Kickstarter-iOS/Views/Controllers/PledgeViewController.swift

```diff
@@ -26,12 +53,153 @@ final class PledgeViewController: UIViewController {
+
+    _ = (self.rootScrollView, self.view)
+      |> ksr_addSubviewToParent()
+      |> ksr_constrainViewToMarginsInParent()
+
+    _ = (self.rootStackView, self.rootScrollView)
+      |> ksr_addSubviewToParent()
+      |> ksr_constrainViewToEdgesInParent()
+
+    self.configureChildViewControllers()
+
+    self.view.addGestureRecognizer(
+      UITapGestureRecognizer(target: self, action: #selector(PledgeViewController.dismissKeyboard))
+    )
+
+    self.setupConstraints()
+    self.viewModel.inputs.viewDidLoad()
+  }
+
+  deinit {
+    self.sessionStartedObserver.doIfSome(NotificationCenter.default.removeObserver)
+  }
+
+  // MARK: - Configuration
+
+  private func configureChildViewControllers() {
+    self.addChild(self.pledgeDescriptionViewController)
+    self.addChild(self.pledgeAmountViewController)
+    self.addChild(self.pledgeShippingLocationViewController)
+    self.addChild(self.pledgeSummaryViewController)
+    self.addChild(self.pledgeContinueViewController)
+    self.addChild(self.pledgePaymentMethodsViewController)
+
+    _ = ([
+      self.pledgeDescriptionViewController.view,
+      self.pledgeAmountViewController.view,
+      self.pledgeShippingLocationViewController.view,
+      self.pledgeSummaryViewController.view,
+      self.pledgeContinueViewController.view,
+      self.pledgePaymentMethodsViewController.view
+    ], self.rootStackView)
+      |> ksr_addArrangedSubviewsToStackView()
+
+    self.pledgeDescriptionViewController.didMove(toParent: self)
+    self.pledgeAmountViewController.didMove(toParent: self)
+    self.pledgeShippingLocationViewController.didMove(toParent: self)
+    self.pledgeSummaryViewController.didMove(toParent: self)
+    self.pledgeContinueViewController.didMove(toParent: self)
+    self.pledgePaymentMethodsViewController.didMove(toParent: self)
+  }
+
+  private func setupConstraints() {
+    NSLayoutConstraint.activate([
+      self.rootStackView.widthAnchor.constraint(equalTo: self.view.layoutMarginsGuide.widthAnchor)
+    ])
+  }
+
+  // MARK: - Styles
+
+  override func bindStyles() {
+    super.bindStyles()
+
+    _ = self.view
+      |> checkoutBackgroundStyle
+
+    _ = self.rootScrollView
+      |> rootScrollViewStyle
+
+    _ = self.rootStackView
+      |> rootStackViewStyle
+  }
+
+  // MARK: - View model
+
+  override func bindViewModel() {
+    super.bindViewModel()
+
+    self.viewModel.outputs.configureWithPledgeViewData
+      .observeForUI()
+      .observeValues { [weak self] data in
+        self?.pledgeDescriptionViewController.configureWith(value: data.reward)
+        self?.pledgeAmountViewController.configureWith(value: (data.project, data.reward))
+        self?.pledgeShippingLocationViewController.configureWith(value: (data.project, data.reward))
+        self?.pledgePaymentMethodsViewController.configureWith(value: [GraphUserCreditCard.template])
+      }
+
+    self.viewModel.outputs.configureSummaryCellWithData
```

Comment:
Rename `configureSummaryCellWithData` to `configureSummaryViewControllerWithData`